addon/components/table.ts

```diff
@@ -197,8 +322,8 @@ class TableComponent extends Component.extend({ ResizeAware }) {
         //fixed columns are disabled if the widest non-fixed column cannot
         //fit in the container at the same time as the fixed column(s)
         if (!isEmpty(this.fixedColumns)) {
-            const sortedColumns = get(this, 'nonFixedColumns').sortBy('staticWidth');
-            const widestColumn = get(sortedColumns, 'lastObject');
+            const sortedColumns = this.nonFixedColumns.sortBy('staticWidth');
+            const widestColumn = sortedColumns.slice(-1)[0];
```

Comment:
`const widestColumn = sortedColumns[sortedColumns.length - 1];` is probably a little better here